.github/workflows/dev-ci-cd.yml

```
@@ -0,0 +1,96 @@
+name: Dev CI/CD - Build and Deploy
+
+on:
+  push:
+    branches: [ develop ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)